cirq/google/engine/engine_timeslot.py

```diff
@@ -13,18 +13,24 @@
 # limitations under the License.
 import dataclasses
 import datetime
-
 from typing import Optional
+from google.protobuf import timestamp_pb2
 from cirq.google.engine.client.quantum_v1alpha1.gapic import enums
```

Comment:
Can we re-export this type somewhere so that it's easier for users to consume (e.g. filter)?

Reply:
Done.